src/main/frontend/modules/shortcut/config.cljs

```diff
@@ -101,6 +102,9 @@
    :editor/new-line              {:binding "shift+enter"
                                   :fn      editor-handler/keydown-new-line-handler}
 
+   :editor/new-whiteboard        {:binding "mod+shift+w"
```

Comment:
I am not sure if `:editor/` is the best option for this action, but :`go` also feels weird, and introducing a new namespace for `whiteboards` seems excessive for a single action.

Reply:
It seems `mod+shift+w` shortcut is the same as "close all tabs" function in a browser. Could you please find another option, or just level it nil?

Reply:
I like the `new` convention approach. `n` `w` feels intuitive

Reply:
Personally, I prefer `n`, `w` because it's fast and easy to remember as the `g` for go. My concern is that unlimited chords could result in conflicts with plugins, for example, the vim plugin might use `n`, tldraw uses `t` for creating a text element.

Reply:
`c` definitely creates a problem because plugins can't register chords that start with `c`. Chords with multiple keys such as `n` `w` might be ok.

Reply:
Switched to `n` `w`.
I also added a comment to discourage the usage of single letter shortcuts.